<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpritePackerClass</class>
 <widget class="QMainWindow" name="SpritePackerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpritePacker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <layout class="QGridLayout" name="gridContainer" rowstretch="3,0,0,0,0,0,0,0" columnstretch="0,0,0,0,0,0,0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <property name="spacing">
       <number>16</number>
      </property>
      <item row="5" column="4" colspan="2">
       <widget class="QListWidget" name="generatedAtlasList"/>
      </item>
      <item row="2" column="2" rowspan="4">
       <widget class="QListWidget" name="framesList"/>
      </item>
      <item row="4" column="4" colspan="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="generatedAtlasLabel">
        <property name="text">
         <string>Generated Atlas</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="clipsListLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Clips</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="framesFilter">
        <property name="placeholderText">
         <string>Filter Frames</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="animationsFilter">
        <property name="placeholderText">
         <string>Filter Animations</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="3">
       <widget class="QComboBox" name="jsonVersionComboBox">
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>1.4.3.2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.2.2.1</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="3" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="originalAtlasLabel">
        <property name="text">
         <string>Original Atlas</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>zh-CN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>en-US</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="6" colspan="2">
       <widget class="QPushButton" name="restoreButton">
        <property name="text">
         <string>Restore</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="4">
       <widget class="QListWidget" name="animationsList"/>
      </item>
      <item row="0" column="8" colspan="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="changedFilesLabel">
        <property name="text">
         <string>Changed Files</string>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="3">
       <widget class="QLabel" name="inspectorLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Inspector</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="animationsListLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Animations</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="8" colspan="2">
       <widget class="QPushButton" name="replaceAllButton">
        <property name="text">
         <string>Replace All</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="clipsFilter">
        <property name="placeholderText">
         <string>Filter Clips</string>
        </property>
       </widget>
      </item>
      <item row="6" column="9">
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="6" column="7">
       <widget class="QPushButton" name="sortButton">
        <property name="text">
         <string>Sort</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="framesListLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Frames</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="6">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Backup</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QListWidget" name="originalAtlasList"/>
      </item>
      <item row="2" column="3" colspan="3">
       <widget class="QPushButton" name="packButton">
        <property name="text">
         <string>Pack</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QPushButton" name="replaceButton">
        <property name="text">
         <string>Replace</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="4">
       <widget class="QListWidget" name="clipsList"/>
      </item>
      <item row="6" column="3" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="languageLabel">
        <property name="text">
         <string>Language</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6" colspan="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="backupFilesLabel">
        <property name="text">
         <string>Backup Files</string>
        </property>
       </widget>
      </item>
      <item row="6" column="8">
       <widget class="QPushButton" name="unwatchButton">
        <property name="text">
         <string>Unwatch</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8" rowspan="5" colspan="2">
       <widget class="QListWidget" name="changedFilesList"/>
      </item>
      <item row="1" column="6" rowspan="5" colspan="2">
       <widget class="QListWidget" name="backupFilesList"/>
      </item>
      <item row="3" column="3" colspan="3">
       <widget class="QLabel" name="spritesView">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="4" colspan="2">
       <widget class="QPushButton" name="checkButton">
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" rowspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="getSingleSpritesLink">
        <property name="text">
         <string>&lt;a href=&quot;https://github.com/magegihk/HollowKnight.GODump&quot;&gt;Get single sprites&lt;/a&gt;</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1" rowspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="aboutSpritePackerLink">
        <property name="text">
         <string>&lt;a href=&quot;https://github.com/magegihk/HollowKnight.SpritePacker&quot;&gt;About Sprite Packer&lt;/a&gt;</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
      <item row="6" column="2" rowspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="joinAndDrawLabel">
        <property name="text">
         <string>Join and Draw!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenSpritesFolder">
   <property name="text">
    <string>Open Sprites Folder</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
